root/artist/edit_form.tt

```diff
@@ -97,16 +101,93 @@
+              It’s okay to leave it blank if it seems unlikely that there will be other artists with similar names.',
+             { doc => { href => doc_link('Disambiguation_Comment'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('This field is not a place to store general background information.
+              That kind of information should go in an {doc|annotation}.',
+             { doc => { href => doc_link('Annotation'), target => '_blank' } }) %]
+      </p>
+    </div>
+
     [%- type_bubble(form.field('type_id'), artist_types) -%]
 
+    <div class="bubble" id="gender-bubble">
+      <p>
+        [% l('The {doc|gender} field describes the gender with which a person or character identifies.',
+             { doc => { href => doc_link('Artist#Gender'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('Please see the {doc|style guidelines} for more information.',
+             { doc => { href => doc_link('Style/Artist#Gender'), target => '_blank' } }) %]
+      </p>
+    </div>
+
+    <div class="bubble" id="area-bubble">
+      <p>
+        [% l('The {doc|area} indicates the area the artist is primarily identified with.',
+             { doc => { href => doc_link('Artist#Area'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('Please see the {doc|style guidelines} for more information.',
+             { doc => { href => doc_link('Style/Artist#Main_area'), target => '_blank' } }) %]
+      </p>
+      [% area_bubble_selection() %]
+    </div>
+
     [%- ipi_bubble() -%]
     [%- isni_bubble() -%]
-  </div>
 
+    <div class="bubble" id="begin-end-date-bubble">
+      <p>
+        [% l('The {doc|begin and end dates} describe when the artist started and stopped existing.',
+             { doc => { href => doc_link('Artist#Begin_and_end_dates'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('Please see the {doc|style guidelines} for more information.',
+             { doc => { href => doc_link('Style/Artist#Begin_and_end_areas/dates'), target => '_blank' } }) %]
+      </p>
+    </div>
+
+    <div class="bubble" id="begin-end-area-bubble">
+      <p>
+        [% l('The begin and end areas describe where the artist was when it started and stopped existing.') %]
+      </p>
+      <p>
+        [% l('Please see the {doc|style guidelines} for more information.',
+             { doc => { href => doc_link('Style/Artist#Begin_and_end_areas/dates'), target => '_blank' } }) %]
+      </p>
+      [% area_bubble_selection() %]
+    </div>
+
+    <div class="bubble" id="external-link-bubble">
+      <p>
+        [% l('External links are URLs associated with the artist.
+              These can include the artist’s official homepage and social networking pages, Wikipedia entries,
+              streaming services, and third‐party pages like biographies, interviews, and fan sites.') %]
```

Comment:
I figured it'd be useful to list some examples of what can be entered here, since I don't think there's any way for editors to see the different link types until they've pasted a URL.

Reply:
Some examples seem fine. Things like interviews seem so niche I'm not sure there's a point in mentioning them, but maybe @Aerozol wants to offer something :)

Reply:
It looks fine to me, I don't mind interviews being included in the list.
I wonder if the "third-party" description is accurate though? I'm not sure what first or second party is in this context. Perhaps it can be left out:

"These can include the artist’s official homepage and social networking pages, Wikipedia entries, streaming services, and pages like biographies, interviews, and fan sites."

My only other thought is that maybe we should have "Wikidata entries" instead of Wikipedia, since we prefer those links, but on reflection I think new users are struggle enough, maybe we shouldn't be making them learn about Wikidata at the same time!

Reply:
Would "unofficial" be clearer than "third-party"? Maybe "fan pages" already makes it obvious that these can be unofficial, though.

Reply:
I've switched this to "unofficial" and moved Wikipedia articles to that section.

Reply:
Pasting @Aerozol's comment here since I'm having trouble finding it through the UI:

> Unofficial might open a can of worms tbh, I quite link the wording of "third-party", which still implies some sort of legitimacy (not links to torrents etc). It's probably fine as-is, I may have gotten fixated on Wikipedia not being "first-party"...

@reosarevok, do you have any opinions about using "third-party" vs. "unofficial" in these strings? Searching through the MB documentation pages, I see "third-party" mostly used to refer to tools not maintained by MeB and "unofficial" mostly used to refer to bootleg releases.

Reply:
I think there's no ideal option, I'm fine with unofficial really, I doubt anyone will overthink it as much as we would and if that happens we can reconsider.

Reply:
Cool, sounds good to me (and luckily, "unofficial" is what this change is currently using). :-)